libs/Communicator/Communicator.hpp

```diff
@@ -34,9 +44,13 @@ namespace communicator_lib
         bool newClient;
     };
 
+    /// @brief Configuration of a room. It contains a name and 6 modifiers.
     struct RoomConfiguration {
+        /// @brief Name of the room.
         std::string roomName;
-        unsigned short configs[6];
+        /// @brief Configuration (6 modifiers).
+        /// Use enumerator roomConfiguration_e to set value at the good index.
+        short configs[6];
```

Comment:
Why is it no longer an unsigned short?

Reply:
It was a missing, I need signed value and not unsigned value